fastplotlib/widgets/image.py

```diff
@@ -372,149 +360,29 @@ def __init__(
                 f"You have passed the following type {type(data)}"
             )
 
-        # default dims order if not passed
-        # updated later if passed
+        if self.ndim not in DEFAULT_DIMS_ORDER.keys():
+            raise ValueError(f"{self.ndim} dimensions not supported")
```

Comment:
add something like "only xy, txy, and tzxy data with or without RGB(A) is supported"